tests/foreman/rhci/test_rhci.py

```diff
@@ -76,26 +78,25 @@ def test_positive_update_name(self, data):
         @Feature: RHCI Deployment
 
         """
-        deployment = entities.RHCIDeployment(
-            name=data['name'],
-            organization=self.org,
-            lifecycle_environment=self.lc_env,
-            deploy_rhev=True,
-            rhev_engine_host=self.rhev_engine_host,
-            rhev_storage_type='NFS',
-            rhev_engine_admin_password='changeme',
-        ).create()
-        self.assertEquals(deployment.name, data['name'])
+        for data in valid_name_update_tests():
```

Comment:
I am not the right person to change the test criteria for `rhci` tests.  My only idea of this PR is to remove @ddt, @data usage so I can eventually remove ddt dependency for robottelo.

If this optimization is required, I will let @anarang do that.